crates/db_schema/src/source/comment_reply.rs

```diff
@@ -0,0 +1,25 @@
+use crate::newtypes::{CommentId, CommentReplyId, PersonId};
+use serde::{Deserialize, Serialize};
+
+#[cfg(feature = "full")]
+use crate::schema::comment_reply;
+
+#[derive(Clone, PartialEq, Debug, Serialize, Deserialize)]
+#[cfg_attr(feature = "full", derive(Queryable, Associations, Identifiable))]
+#[cfg_attr(feature = "full", belongs_to(crate::source::comment::Comment))]
+#[cfg_attr(feature = "full", table_name = "comment_reply")]
+pub struct CommentReply {
```

Comment:
This is to keep track of comments whiuch are marked as read by the recipient, right? Could you add a comment to explain that? Its not so obvious.

Reply:
Can do.